src/main/java/io/wispforest/worldmesher/WorldMesh.java

```diff
@@ -156,6 +156,13 @@ public Box dimensions() {
         return dimensions;
     }
 
+    public void reset() {
```

Comment:
I'd like this method to be called `clear()` instead. Ideally you'd also add some short Javadoc explaining what it does and why it might be needed. Other than than, looks good